convert.py

```diff
@@ -25,7 +25,7 @@
 for f in source_files:
     subprocess.call(["inkscape", f, "--export-plain-svg=generated/" + os.path.splitext(f)[0] + ".svg"])
 
-subprocess.call(["webicon.sh", "sunpy_icon.svg"])
+subprocess.call(["./webicon.sh", "sunpy_icon.svg"])
```

Comment:
wouldn't be better instead to use: 
```python
subprocess.call(["bash", "webicon.sh", "sunpy_icon.svg"])
```
so it runs even if the permissions are not set properly?